docs/ensembl-help/using-ensembl/ensembl-apps/blast/blast.md

```diff
@@ -0,0 +1,66 @@
+---
+slug: blast
+title: How to run BLAST
+description: How to run BLAST queries
+related_articles:
```

Comment:
I think you need to remove this line - the build has failed at this point. I suspect it can't be empty.

Reply:
Thanks for spotting Anne. This is done now